x-pack/plugin/watcher/src/main/java/org/elasticsearch/xpack/watcher/trigger/schedule/engine/TickerScheduleTriggerEngine.java

```diff
@@ -81,17 +85,22 @@ public synchronized void start(Collection<Watch> jobs) {
 
     @Override
     public void stop() {
+        logger.debug("Stopping engine");
+        isRunning.set(false);
         schedules.clear();
         ticker.close();
     }
 
     @Override
-    public synchronized void pauseExecution() {
+    public void pauseExecution() {
+        logger.debug("Pausing engine");
```

Comment:
might want to make these engine state changes stop/start/pause at info level. Debug can be very verbose, and I think these are worthy of info.